x-pack/plugins/infra/public/components/asset_details/tabs/overview/tile.tsx

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
I'm wondering what we could do to avoid duplicating this component. We should probably have a common folder. Not sure if that would work once we move the asset details to another plugin. But might be worth exploring the possibilities.

Reply:
Yeah, there are other dependencies still (`LensWrapper`, `buildCombinedHostsFilter`, `TooltipContent`) and they are reused in other places. The host filter build helper is not really asset details specific and we can't move it because it should also live in `infra` in the future (so when the component lives outside of `infra` because we also don't want to get it from `infra` once we move the asset details).
I will first pull the latest `main` here and I will think about how to move more of the shared parts ( and at least for now we can reuse them from a common folder)

Reply:
I moved the dependencies and checked the component and there are still some differences in the filters, also the host component is connected to the host state while in asset view I changed it to accept props for the `nodeName`, `dateRange`, and `dataView`. We can also have a common component and 2 wrappers to pass the data but then it's still something we need to rework when we move it, should we do that and have 3 components - one with the common parts and 2 wrappers (1 in asset details and 1 in host page)?
If we keep it in the asset details as a separate one we are also flexible to change it without affecting the host page. wdyt?